index.html

```diff
@@ -0,0 +1,170 @@
+          </div>
+        </div>
+        <div class="grid-container">
+          <div class="main-program-card">
+            <div class="card-icon">
+              <img src="images/comment.png" alt="forum">
+            </div>
+            <div class="main-program-title">
+              <h2>Forum</h2>
+            </div>
+            <div class="main-program-text">
+              <p>listen to the speakers from various countries about the message of sharign and opening</p>
+            </div>
+          </div>
+        </div>
+        <div class="grid-container">
+          <div class="main-program-card">
+            <div class="card-icon">
+              <img src="images/workshop.png" alt="">
+            </div>
+            <div class="main-program-title">
+              <h2>Workshop</h2>
+            </div>
+            <div class="main-program-text">
+              <p>listen to the speakers from various countries about the message of sharign and opening</p>
+            </div>
+          </div>
+        </div>
+        <div class="grid-container">
+          <div class="main-program-card">
+            <div class="card-icon">
+              <img src="images/ignite.png" alt="ignite">
+            </div>
+            <div class="main-program-title">
+              <h2>Fireworks</h2>
+            </div>
+            <div class="main-program-text">
+              <p>listen to the speakers from various countries about the message of sharign and opening</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="join-btn">
+        <button class="join-concert-button" type="button">Join CC global Summut 2022</button>
+      </div>
+    </section>
+    <section class="featured-section">
+      <div class="featured-head-speaker"> 
+        <h2>Featured DJ's</h2>
+        <hr class="underline">
+      </div>
+      <div class="speaker-wrapper" id="speaker-wrapper">
+      </div>
+      <div class="more-btn">
+        <button class="more-section-button" id="toggleButton" onclick="toggle()" type="button">MORE <i class="gg-arrows-scroll-v"></i></button>
+      </div>
+    </section>
+    <section class="partner-section">
+      <div class="partner-section-wrapper">
+        <div class="featured-head"> 
+          <h2>Partners</h2>
+          <hr class="underline-partner">
+        </div>
+        <div class="partners-img">
+          <img class="sandbox-img" src="images/nike.png" alt="Logo">
+          <img class="viking-img" src="images/airbnb.png" alt="Logo">
+          <img class="sandbox-img" src="images/nike.png" alt="Logo">
+          <img class="viking-img" src="images/airbnb.png" alt="Logo">
+          <img class="sandbox-img" src="images/nike.png" alt="Logo">
+        </div>
+        <hr>
+      </div>
+    </section>
+    <footer>
+      <div class="footer-wrappter">
+        <img src="images/footer-img.png" alt="creative-common">
+        <p>Tommorowland concert 2022 Some Rights Reserved</p>
+    </div>
```

Comment:
- [ ] Please the logo must be aligned to the left and text aligned to the right. I suggest you adjust `gap` to fix that
-
![Capture](https://user-images.githubusercontent.com/63972670/180994090-f7b362df-6d0e-4a5e-9f0a-8044ce5a4caf.PNG)